eZ/Publish/Core/settings/comparable_fieldtypes.yml

```diff
@@ -0,0 +1,55 @@
+services:
+    eZ\Publish\Core\Compare\TextCompareEngine:
```

Comment:
`TextComparisonEngine`, assuming there will be some indication in the namespace that this is related to Content and/or Versions.